blog/_posts/2020-04-02-march-commit-of-the-month.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: march-2020-commit-of-the-month
+date: 2020-04-02
+author: dmiller
+layout: blog
+title: "Accessibility Matters"
+subtitle: "Commit of the Month: Improve Accessibility of Status Icons"
+image: /assets/images/march-2020-commit-of-the-month/featuredImage.jpg
+image_caption: "A colorblind test. Photo by daltonien on <a href='https://flickr.com/photos/nicolasdumond79/13782659973'>Flickr</a>."
+image_type: "contain"
+tags:
+  - tilt
+  - cotm
+  - accessibility
+keywords:
+  - tilt
+  - accessibility
+---
+
+Details matter. Especially when those details impact usability. That’s why we paid close attention when we got reports that Tilt’s terminal UI was not colorblind friendly. Take a look at this screenshot:
+
+![A photo of the Tilt UI with green and red icons next to each resource](/assets/images//march-2020-commit-of-the-month/colorblind1.png)
+
+Can you tell at a glance which of these resources are up, and which ones have errored? It’s hard. If you were red-green colorblind it would be even harder. It might look something like this:
+
+![A photo of the Tilt UI with green and red icons next to each resource modified to show what it might look like to a colorblind person](/assets/images/march-2020-commit-of-the-month/colorblind1-mod.png)
+
+While we expect most people to be using the Web UI (which is more fully featured and happened to not exhibit this problem by dint of a different design) if your workflow lives in the terminal we want to support that too. So in [842b83131f78431dd1ba8522e6288226a09c4cfc](https://github.com/windmilleng/tilt/commit/842b83131f78431dd1ba8522e6288226a09c4cfc) we improved the terminal UI’s accessibility by adding status icons that are structurally distinct from one another in addition to having distinct hues. Now Tilt’s status icons look like this:
```

Comment:
this text isn't quite accurate. The WebUI had this problem when the user reported it. We fixed it in a different way in a different PR https://github.com/windmilleng/tilt/pull/2955/

Reply:
Ah I see, thanks! I looked for something like that but missed it the first time. Should be fixed now.